types.d.ts

```diff
@@ -1,6 +1,6 @@
 declare namespace PgBoss {
   interface Db {
-    executeSql(text: string, values: any[]): Promise<{ rows: any[]; rowCount: number }>;
+    executeSql(text: string, values: any[], options?: ExecutionOptions): Promise<{ rows: any[]; rowCount: number }>;
```

Comment:
Maybe this `?` can be removed. It looks like `options` will always be defined based on how its called.

Reply:
There's a few places where executeSql is called without any values too: https://github.com/timgit/pg-boss/blob/master/src/contractor.js#L26

I'm not super familiar with typescript, is that valid?

Reply:
My bad! I assumed `executeSql` was only called there